.changelog/801.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Update hcp_iam_policy docs with HCP Vault Secrets Roles App Manager and App Secret Reader
```

Comment:
```suggestion
Documentation: Update hcp_iam_policy docs with HCP Vault Secrets Roles App Manager and App Secret Reader
```

Reply:
Just from looking at other changelogs, I think documentation updates use the `Documentation:` prefix.